src/Turing.jl

```diff
@@ -18,6 +18,10 @@ import StatsBase
 import Printf
 import Random
 
+using ADTypes: ADTypes
+
+default_adtype() = ADTypes.AutoForwardDiff()
```

Comment:
Wouldn't a constant be simpler?
```suggestion
const DEFAULT_ADTYPE = ADTypes.AutoForwardDiff()
```

Reply:
Actually did that first :sweat_smile: 

Buuuut using a method _does_ allow an advanced user to actuallly overload the default AD backend that is used :shrug:

Reply:
I'm not completely convinced that this should be supported since it has such widereaching consequences. Some limited flexibility could be allowed by making the default a Preferences-constant that could be modified by users.